filebeat/module/iis/access/test/test.log-expected.json

```diff
@@ -1,105 +1,105 @@
 [
     {
         "@timestamp": "2018-01-01T08:09:10.000Z", 
+        "destination.ip": "127.0.0.1", 
+        "destination.port": "80", 
         "event.dataset": "access", 
         "event.module": "iis", 
-        "iis.access.geoip.city_name": "Berlin", 
-        "iis.access.geoip.continent_name": "Europe", 
-        "iis.access.geoip.country_iso_code": "DE", 
-        "iis.access.geoip.location.lat": 52.4908, 
-        "iis.access.geoip.location.lon": 13.3275, 
-        "iis.access.geoip.region_iso_code": "DE-BE", 
-        "iis.access.geoip.region_name": "Land Berlin", 
-        "iis.access.method": "GET", 
-        "iis.access.port": "80", 
-        "iis.access.query_string": "q=100", 
-        "iis.access.referrer": "-", 
-        "iis.access.remote_ip": "85.181.35.98", 
+        "http.request.method": "GET", 
```

Comment:
We should probably add a note to ECS that this should always be upper case.

Reply:
Good idea

Reply:
Added a note to the planning. Resolving this comment